canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -230,13 +230,12 @@ $palette-border-color: $ui-03;
 }
 
 .palette-flyout-category-text {
-	line-height: 46px;  /* Must be same as containing div height */
-	overflow: hidden;
-	text-overflow: ellipsis;
-	white-space: nowrap;
+	display: flex;
```

Comment:
Why dos display need to be flex? If I take it out it seems to display OK. Please don't add fields into the CSS unless they are necessary.

Reply:
you are right !! I was trying flex-wrap in the first place and get elements gets wrapped  however it didn't work..white-space: nowrap did the job. so I had to remove flex.. hmmm let check what is the issue with caret icon